core/src/program/info.rs

```diff
@@ -0,0 +1,68 @@
+use super::{ByteWriter, Digest, Kernel, Program, Serializable};
+
+// PROGRAM INFO
+// ================================================================================================
+
+/// A program information set consisting of its code block root and set of kernel roots used for
```

Comment:
nit: I'd change "code block root" to "MAST root" and "kernel roots" to "kernel procedure roots".

Reply:
Done!